src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -123,27 +124,18 @@ private void initUI() {
         VBox milestoneDialogBox = createMilestoneDialogBox();
         assignedMilestoneBox = createAssignedMilestoneBox();
         matchingMilestonesBox = createMatchingMilestonesBox();
-        ScrollPane matchingMilestonesScrollPane = coverWithScrollPane(matchingMilestonesBox);
         inputField = createInputField();
 
-        milestoneDialogBox.getChildren().addAll(assignedMilestoneBox, inputField, matchingMilestonesScrollPane);
+        milestoneDialogBox.getChildren().addAll(assignedMilestoneBox, inputField, matchingMilestonesBox);
```

Comment:
Did you remove the scroll pane because now it should show only 5 best matching milestones? So even if input field is empty, it will still just show 5 milestones?